phantasm-at-rest/src/test/java/com/chiralbehaviors/CoRE/phantasm/graphQl/WorkspaceSchemaTest.java

```diff
@@ -61,12 +63,37 @@
     private static final String TEST_SCENARIO_URI = "uri:http://ultrastructure.me/ontology/com.chiralbehaviors/demo/phantasm/v1";
 
     @Test
-    public void testGraphQlResource() throws Exception {
+    public void testBadQueries() throws Exception {
```

Comment:
This is the test that validates unbound variables doesn't blow up and the execution returns a useful validation error, correctly typed and with a useful message.